scripts/build.mts

```diff
@@ -104,14 +107,15 @@ for await (const file of filesTreeGenerator(schemasFolder)) {
     },
   });
 
-  await fsPromise.appendFile(`${fileDestPath}.ts`, '\nconst schema = ' + JSON.stringify(schema) + 'as const;\n', { encoding: 'utf-8' });
-  await fsPromise.appendFile(
-    `${fileDestPath}.ts`,
-    'export type schemaType = FromSchema<typeof schema, {parseIfThenElseKeywords: true, parseNotKeyword: true}>;',
-    {
-      encoding: 'utf-8',
-    }
-  );
+  const schemaTs = 'const schema = ' + JSON.stringify(dereferencedSchema) + 'as const;';
```

Comment:
Shouldn't it be `' as const;'`?

Reply:
changed, but its not very important as the code is transpiled to js afterwards.